sys-libs/libexecinfo/libexecinfo-1.1.recipe

```diff
@@ -6,7 +6,7 @@ HOMEPAGE="https://www.freshports.org/devel/libexecinfo"
 COPYRIGHT="2003-2014 Maxim Sobolev"
 LICENSE="BSD (2-clause)"
 REVISION="6"
-SOURCE_URI="https://sources.voidlinux-ppc.org/libexecinfo-$portVersion/libexecinfo-$portVersion.tar.bz2"
+SOURCE_URI="http://distcache.freebsd.org/local-distfiles/itetcu/libexecinfo-$portVersion.tar.bz2"
```

Comment:
Previous download produces random checksums, this was the closest available I could find.